### ISSUE TITLE
Web site - broken links

### ISSUE THREAD

**Issue by IGotNothin** [Json2CSharp/Json2CSharpCodeGenerator#124]
Super minor but there's various links on your web site that either don't work or are going to the wrong place. 

1. Header -> Blog - site doesn't exist.
2. Footer -> Xamasoft link - redirects to something my browser blocks, so maybe their domain is no longer theirs.
3. Privacy Policy page - Github and Report an Issue links don't work. 
4. Footer -> header links (Code Converters, CSS Tools, etc.) - linked to invalid pages. 

I was actually just looking for an "about us" or "support us" kind of page as I find the site to be very valuable.  Thanks for your work!

**Comment by ghost** [comment]
Yeah the site is a bit rusty, but a development plan is on the way, thinking of including an AI assistant to help with some transformation. It should be tailored and properly integrated for it to make sense, otherwise anyone can just copy the content to chatgpt
